scripts/install_xcpng.py

```diff
@@ -155,7 +167,9 @@ def main():
         help="hostname or IP address of the host hosting the VM that will be installed/upgraded"
     )
     parser.add_argument("vm_uuid", help="UUID of an existing VM in which XCP-ng will be installed (or upgraded)")
-    parser.add_argument("action", metavar='action', choices=['install', 'upgrade'], help="install or upgrade")
+    parser.add_argument(
+        "action", metavar='action', choices=['install', 'upgrade', 'restore'], help="install, upgrade or restore"
+    )
     parser.add_argument(
         "xcpng_version",
         help="target version, used to build the installer URL if none provided via --installer, "
```

Comment:
The help message needs to be completed with what the version means in the case of a restore.